### PR TITLE
#202 Override gas estimator values so messages can be mined on first try

### PR DESCRIPTION
**Pull request by aistoc**

### CODE REVIEW

File: src/shared/lotus-client/Lotus.ts [hunk 1]
```diff
@@ -92,12 +94,12 @@ export class Lotus {
 
         //appStore.logsStore.logDebug(`Lotus.gesGasEstimation response:\n ${inspect(response.data)}\n`);
 
-        message.gaslimit = response.data.result.GasLimit * 2
-        message.gasfeecap = response.data.result.GasFeeCap
+        message.gaslimit = response.data.result.GasLimit * 2.4
+        message.gasfeecap = minCurrentBaseFee //response.data.result.GasFeeCap
```
Comment (brunolm) [inline src/shared/lotus-client/Lotus.ts:98]:
```suggestion
        message.gasfeecap = Math.max(minCurrentBaseFee, response.data.result.GasFeeCap)
```

Comment (brunolm) [inline src/shared/lotus-client/Lotus.ts:98]: Need to use `Math.max`, but if it's a BigNumber we have to use a function that works with BigNumber

File: src/shared/lotus-client/Lotus.ts [hunk 2]
```diff
@@ -92,12 +94,12 @@ export class Lotus {
 
         //appStore.logsStore.logDebug(`Lotus.gesGasEstimation response:\n ${inspect(response.data)}\n`);
 
-        message.gaslimit = response.data.result.GasLimit * 2
-        message.gasfeecap = response.data.result.GasFeeCap
+        message.gaslimit = response.data.result.GasLimit * 2.4
```
Comment (brunolm) [inline src/shared/lotus-client/Lotus.ts]:
```suggestion
        message.gaslimit = response.data.result.GasLimit * 2
```

File: src/shared/lotus-client/Lotus.ts [hunk 3]
```diff
@@ -92,12 +94,12 @@ export class Lotus {
...
+        message.gasfeecap = minCurrentBaseFee //response.data.result.GasFeeCap
         message.gaspremium = response.data.result.GasPremium
 
       } catch (error) {
-        message.gaslimit = parseInt(gasEstimation.gaslimit) * 2
+        message.gaslimit = parseInt(gasEstimation.gaslimit) * 2.4
```
Comment (brunolm) [inline src/shared/lotus-client/Lotus.ts]:
```suggestion
        message.gaslimit = parseInt(gasEstimation.gaslimit) * 2
```

File: src/shared/lotus-client/Lotus.ts (hunk 1 above)
Comment (brunolm) [inline src/shared/lotus-client/Lotus.ts]: Need to use Math.max, but if it's a BigNumber we have to use a function that works with BigNumber

Comment (Andrel1s) [reply src/shared/lotus-client/Lotus.ts:98]: response.data.result.GasFeeCap

**Pull request merged**